### PR TITLE
Created Button Components and Page Structure

### PR DESCRIPTION
**Pull request by Sharmaa-Ansh**

### CODE REVIEW

File: Components/Button/Button.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                Want to attach a link action, at a button tap? Use Link Button.
+            </div>
+
+            <div class="ExampleBox">
+                <a href="Link" class="btn btn--Primary-Link">Link </a>
+            </div>
+            <div class="Code-Snippet-Display">
+                <iframe
+                src="https://carbon.now.sh/embed/syBzxbr7wFRNlS2Cgo46"
+                style="width: 730px; height: 111px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (romabulani) [inline Components/Button/Button.html]: Avoid inline style tags

File: Components/Button/Button.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            </div>
+
+            <div class="ExampleBox">
+                <a href="Link" class="btn-Float"></a>
+               <i class="fas fa-plus btn-Float-Icon"></i>
+            </div>
+            <div class="Code-Snippet-Display">
+                <iframe
+                src="https://carbon.now.sh/embed/syBzxbr7wFRNlS2Cgo46"
+                style="width: 730px; height: 111px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (romabulani) [inline Components/Button/Button.html]: Here also dont use inline style tag

File: Styles.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+.body-container{
+    display: flex;
+    max-width: 90%;
+    margin: auto;
+}
+
+.Code-Snippet-Display{
+    box-sizing: border-box;
+    width: 100%;
+    max-height: 300px;
```
Comment (romabulani) [inline Styles.css]: use rems instead of px

File: css/sidebar.css [hunk 4]
```diff
@@ -0,0 +1,6 @@
+.sidebar{
+    border: solid black;
+    width: 280px;
```
Comment (romabulani) [inline css/sidebar.css]: use rems

File: Components/Button/Button.css [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    text-decoration: none;
+    padding: 1.5rem;
+    border-radius: 5rem;
+}
+
+.btn-Float-Icon{
+    position: relative;
+    right: 2.8rem;
+    color: white;
+}
```
Comment (romabulani) [inline Components/Button/Button.css]: In this file, at many places white hardcoded is used as color, instead declare variable for it and use var here

**Review by romabulani — COMMENTED** [review]
- Dont use inline style
- Change px to rem
- var for ehite color

File: Components/Button/Button.html (hunk 1 above)
Comment (Sharmaa-Ansh) [reply Components/Button/Button.html]: They come while copying the iframe itself from the carbon sh website.

**Review by shrey27 — APPROVED** [review]

File: Components/Button/Button.html (hunk 1 above)
Comment (romabulani) [reply Components/Button/Button.html]: Ok, understood

**Review by romabulani — APPROVED** [review]
good to merge

**Pull request merged**